package.json

```diff
@@ -5,14 +5,13 @@
   "description": "Morpheus is private, sovereign, AI",
   "main": ".webpack/main",
   "scripts": {
-    "preinstall": "node .setup/yarn-preinstall-system-validation.mjs",
```

Comment:
This solution was added in the following PR: https://github.com/MorpheusAIs/Lite-Client/pull/59
It addressed the following issue: https://github.com/MorpheusAIs/Lite-Client/issues/58
I tested these changes with python 3.12.2 and was able to `start`, `package`, and `make`.